commands/eureka-tasks.js

```diff
@@ -16,14 +16,11 @@ tasks.getTasks()
     })
 
     for (const task of response) {
-      let durationString = ''
-      if (task.durationInSeconds) {
-        const duration = moment.duration(task.durationInSeconds, 'seconds')
-        durationString = sprintf('%4d:%02d:%02d', Math.floor(duration.asHours()), duration.minutes(), duration.seconds())
-      }
-      const costString = task.costInCents ? sprintf('$ %6.2f', task.costInCents / 100.0) : ''
+      const duration = moment.duration(task.durationInSeconds || 0, 'seconds')
+      const durationString = sprintf('%4d:%02d:%02d', Math.floor(duration.asHours()), duration.minutes(), duration.seconds())
```

Comment:
Why `Math.floor(duration.asHours())` and not simply `duration.hours()`? As far as I understand it will extract only the "hour part" of the duration. It is exactly what you are after here.